Makefile

```diff
@@ -46,18 +47,21 @@ OPENAPI_CONFIG:=oas3.config.json
 PACKAGE_NAME=equinix_metal
 PACKAGE_VERSION=0.0.1
 
+USER_AGENT=${PACKAGE_NAME}-${PACKAGE_VERSION}
```

Comment:
I think `metal-python/${PACKAGE_VERSION}` would be preferable here; User-Agent tokens generally use `/` as the separator between name & version, and it'll be helpful to see the language name in our usage reports.

Reply:
@ctreatma thanks for the comment, I fixed it.